R/endpoints.R

```diff
@@ -1,16 +1,20 @@
-#' fetch AFHSB data (point data, no min/max)
+#' Fetch AFHSB data (point data, no min/max)
 #'
-#' API docs: https://cmu-delphi.github.io/delphi-epidata/api/afhsb.html
+#' @param auth Character. Authentication token.
+#' @param locations Character vector. Locations to fetch (region/state, or 3-letter country code labels).
+#' @param epiweeks Epirange. Epiweeks to fetch in the form epirange(startweek, endweek), where
```

Comment:
suggestion: link to `epirange` on first mention.  Like
```
[`epirange`]
```
if you don't care about capitalization.